java-squid/src/main/java/org/sonar/java/resolve/Resolve.java

```diff
@@ -672,6 +679,9 @@ boolean isInheritedIn(JavaSymbol symbol, JavaSymbol.TypeJavaSymbol clazz) {
       case Flags.PUBLIC:
         return true;
       case Flags.PRIVATE:
+        if (symbol.name.equals("<init>")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "<init>" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1132)